.github/workflows/checkstyle.yaml

```diff
@@ -6,39 +6,48 @@ on:
 
 jobs:
   checkstyle:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
       with:
         ref: ${{ github.event.pull_request.head.sha }}
     - name: Install dependencies
       run: |
-        sudo apt-get update
-        sudo apt-get install --yes -qq build-essential autoconf libtool gawk alien fakeroot linux-headers-$(uname -r)
-        sudo apt-get install --yes -qq zlib1g-dev uuid-dev libattr1-dev libblkid-dev libselinux-dev libudev-dev libssl-dev python3 python3-dev python3-setuptools python3-cffi
-        # packages for tests
-        sudo apt-get install --yes -qq parted lsscsi ksh attr acl nfs-kernel-server fio
-        sudo apt-get install --yes -qq mandoc cppcheck pax-utils devscripts
-        sudo -E pip --quiet install flake8
+        sudo -E bash -c '
+          set -e
+          apt-get update
+          xargs --arg-file=${{ github.workspace }}/.github/workflows/build-dependencies.txt apt-get install -qq
```

Comment:
Why do we need one bash run instead of `sudo` on each line, as it was before (and still is in other workflows)? `sudo` per command looks a little bit more readable and uniform to other commands to me.

Reply:
Workflows lack `run-in-privileged-context` feature for a step. It was my attempt at emulate that. I agree it looks little bit odd and is inconsistent with other workflows. On the other I don't think repeating `sudo` in every line is much more elegant.
Since we are entering the area where "Parkinson sends his regards" I'll move it back to the original version.